### DISCUSSION TITLE
How to stop super long file names, and just use header titles?

### DISCUSSION THREAD

**Discussion by DominikMendel** [alphapapa/org-super-agenda#265] [Q&A]
I have an org ref capture that generates a very long name with a duplicated named TODO. 

For example:
Some_long_name.org
TODO Some long name 

But my org agenda view always prefixes with `FILE NAME - TODO NAME` and it's an unreadable underscore format. And very long. 

For example:
Some_long_name.org - [Screen cut off] TODO Some long name 

I want to either format the file name better, or cap the size of the file name to like 10 characters. 
I've played around with `org-agenda-prefix-format` and it seems no matter what I do I can't format the file name or shorten it. 

Any help would be greatly appreciated <3 I've battled with this issue for a while and gave up multiple times trying to fix.

**Comment by alphapapa** [comment]
> I've played around with `org-agenda-prefix-format` and it seems no matter what I do I can't format the file name or shorten it.
> 
> Any help would be greatly appreciated <3 I've battled with this issue for a while and gave up multiple times trying to fix.

That is not related to org-super-agenda.  Please ask in an appropriate forum, like the Org mailing list or https://old.reddit.com/r/orgmode
